docs/docs/cmd/spo/list/list-remove.mdx

```diff
@@ -22,6 +22,9 @@ m365 spo list remove [options]
 `-t, --title [title]`
 : Title of the list to remove. Specify either `id` or `title` but not both.
 
+`--recycle`
+: Recycle the list
```

Comment:
This is not the description provided in the GitHub issue.

Reply:
Updated